examples/amp-position-observer.amp.html

```diff
@@ -94,15 +94,7 @@
     </script>
   </amp-animation>
   <main>
-    <h1>amp-position-observer (Experimental)</h1>
-    <h1><div>Please ensure <pre>amp-position-observer</pre> experiment flag is enabled first</h1> Running
-      <pre>AMP.toggleExperiment('amp-position-observer', true);</pre> in Developer Console will enable the experiment for your browser.
-    </div>
     <div class="spacer"></div>
-
-    <h2>Scrollbound animations with amp-position-observer and amp-animation</h2>
```

Comment:
Was this removed on purpose or accidentally?

Reply:
good catch, fixed.